v2/terraform/ovh/cloud-init.yaml

```
@@ -0,0 +1,151 @@
+#cloud-config
+package_update: true
+package_upgrade: true
+
+packages:
+  - curl
+  - wget
+  - git
+  - unzip
+  - software-properties-common
+  - apt-transport-https
+  - ca-certificates
+  - gnupg
+  - lsb-release
+  - ufw
+  - unattended-upgrades
+
+users:
+  - name: avalanche
+    groups: [sudo, docker]
+    shell: /bin/bash
+    ssh_authorized_keys:
+      - ${ssh_public_key}
+
+# Install Docker
+runcmd:
+  # Install Docker
+  - curl -fsSL https://get.docker.com -o get-docker.sh
+  - sh get-docker.sh
+  - usermod -aG docker avalanche
+  
+  # Install Docker Compose
+  - curl -L "https://github.com/docker/compose/releases/latest/download/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+  - chmod +x /usr/local/bin/docker-compose
+  
+  # Install Nginx
+  - apt install -y nginx certbot python3-certbot-nginx
+  
+  # Create application directory
+  - mkdir -p /opt/avalanche-reporter
+  - chown avalanche:avalanche /opt/avalanche-reporter
+  
+  # Clone repository (change this URL to your repository)
+  - cd /opt/avalanche-reporter
+  - git clone https://github.com/Mik-Grzeg/avalancho.git .
+  - chown -R avalanche:avalanche /opt/avalanche-reporter
+  
+  # Copy environment file
+  - cp env.example .env
+  - chown avalanche:avalanche .env
+  
+  # Configure firewall
+  - ufw default deny incoming
+  - ufw default allow outgoing
+  - ufw allow ssh
+  - ufw allow 80
+  - ufw allow 443
+  - ufw allow 3000
+  - ufw allow 8000
+  - ufw --force enable
+  
+  # Set up automatic security updates
+  - dpkg-reconfigure -plow unattended-upgrades
+  
+  # Create systemd service
+  - |
+    cat > /etc/systemd/system/avalanche-reporter.service << 'EOF'
+    [Unit]
+    Description=Avalanche Reporter Application
+    Requires=docker.service
+    After=docker.service
+
+    [Service]
+    Type=oneshot
+    RemainAfterExit=yes
+    WorkingDirectory=/opt/avalanche-reporter
+    ExecStart=/usr/local/bin/docker-compose -f docker-compose.prod.yaml -f airflow/docker-compose.yaml up -d
```

You could remove `-d` flag to run airflow in the foreground and restart on exit. But maybe you have that already handled by docker-compose.
```suggestion
    ExecStart=/usr/local/bin/docker-compose -f docker-compose.prod.yaml -f airflow/docker-compose.yaml up
```